Kconfig

```diff
@@ -935,7 +935,7 @@ menu "LVGL configuration"
 			default y if !LV_CONF_MINIMAL
 		config LV_USE_OBSERVER
```

Comment:
The Observer shouldn't be among the widgets. Could you move it to the [Others section](https://github.com/lvgl/lvgl/blob/master/Kconfig#L1183)?

Reply:
Done